deps/picotls/t/util.h

```diff
@@ -124,10 +124,27 @@ static inline void setup_session_file(ptls_context_t *ctx, ptls_handshake_proper
     }
 }
 
-static inline void setup_verify_certificate(ptls_context_t *ctx)
+static X509_STORE * openssl_init_cert_store(char const * crt_file)
+{
+    int ret = 0;
+    X509_STORE *store = X509_STORE_new();
+
+    if (store != NULL) {
+        X509_LOOKUP *lookup = X509_STORE_add_lookup(store, X509_LOOKUP_file());
+        ret = X509_LOOKUP_load_file(lookup, crt_file, X509_FILETYPE_PEM);
+        if (ret != 1) {
+            fprintf(stderr, "Cannot load store (%s), ret = %d\n",
+                crt_file, ret);
```

Comment:
Exit here also.

Reply:
Need to handle "crt_file==NULL" case.

Reply:
updated, thanks. when there is error, return NULL. The caller will try default store.

Reply:
updated the change using exit(1).